templates/web/pages/product/includes/edit_history.tt.html

```diff
@@ -8,7 +8,7 @@ <h2 id="history">[% lang('history') %]</h2>
   <li>
     [% revision.date %] - [% display_editor_link(revision.userid) %] [% revision.diffs %] [% IF
     revision.comment %] - [% revision.comment %] [% END %] -
-    <a href="[% this_product_url %]?rev=[% revision.number %]">[% lang('view') %]</a>
+    <a href="[% this_product_url %]?rev=[% revision.number %]">[% lang('View this revision') %]</a>
```

Comment:
New strings need to be added to po/common/common.pot and po/common/en.po , otherwise nothing will be displayed.

It's best to try your changes locally to see the result. You can edit a product on your local product opener to see the history at the bottom.